src/style.css

```diff
@@ -1 +1,66 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Zen+Antique+Soft&display=swap");
```

Comment:
```suggestion
```

add fonts to the <head>

Reply:
the style tags is banned (linthtml). How i can add fonts to the <head>?  or did I misunderstand?

Reply:
you need to add fonts to the HTML into the tag `<head>`
https://mate.academy/learn/css-basics/colors-and-fonts/theory?section=practice